conda/cli/main_export.py

```diff
@@ -11,13 +11,86 @@
     _SubParsersAction,
 )
 
-from ..common.configuration import YAML_EXTENSIONS
+from ..common.constants import NULL
+from ..core.prefix_data import PrefixData
 from ..exceptions import CondaValueError
 
 
+def _create_environment_from_prefix(prefix, env_name, args):
```

Comment:
Future idea (which could apply now, but I won't block on that; I'm just worried we reimplement this again accidentally because we didn't expect it to live in `main_export`):

I think this function might be used in more places than `export`, so at the very least I would move it to `conda.cli.common`, and instead of accepting `args`,  I would write each kwarg explicitly.

Also I'm not sure if it (or a subset of) would make more sense as a `classmethod` in the `Environment dataclass` directly, or even a `to_environment_model()` method in `PrefixData`... but we can always do that later.